src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue25889.cs

```diff
@@ -0,0 +1,25 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue25889 : _IssuesUITest
+	{
+		public Issue25889(TestDevice device) : base(device)
+		{
+		}
+
+		public override string Issue => "CollectionView RemainingItemsThresholdReachedCommand not Firing";
+
+		[Test]
+		[Category(UITestCategories.CollectionView)]
+		public void RemainingItemsThresholdReachedCommandFired()
```

Comment:
The test is failing on iOS:
```
Assert.That(label.GetText(), Is.EqualTo("Command Fired!"))
Expected string length 14 but was 9. Strings differ at index 0.
Expected: "Command Fired!"
But was:  "Not fired"
```

Reply:
I've updated the code based on your feedback. Could you please review it and let me know if there are any further concerns